platform-jobs/samza/auto-creator/src/main/java/org/ekstep/jobs/samza/util/ContentUtil.java

```diff
@@ -64,6 +64,12 @@ public void process(String channelId, String identifier, String repository, Map<
 		String contentStage = "";
 		String internalId = "";
 		Boolean isPublished = false;
+		// Added by tanmay as pkgVersion can also be an integer value
+		// start
+		if(metadata.get(AutoCreatorParams.pkgVersion.name()) != null &&  metadata.get(AutoCreatorParams.pkgVersion.name()) instanceof Integer) {
+			metadata.put(AutoCreatorParams.pkgVersion.name(),String.valueOf(metadata.get(AutoCreatorParams.pkgVersion.name())));
+		}
+		// end
 		Double pkgVersion = Double.parseDouble((String) metadata.getOrDefault(AutoCreatorParams.pkgVersion.name(), "0.0"));
```

Comment:
Convert it to a `Number` object and call use `doubleValue` method from Number object.

Reply:
it is already converting pkdVersion to double if the values is a String
so if we are getting an integer value , just convert to String and it will be converted to double with the already written code

Reply:
resolved